zerver/webhooks/stripe/view.py

```diff
@@ -73,31 +86,33 @@ def topic_and_body(payload: Dict[str, Any]) -> Tuple[str, str]:
 
     # Set the topic to the customer_id when we can
     topic = ""
-    customer_id = object_.get("customer", None)
-    if customer_id is not None:
+    customer_id = object_.get("customer").tame(check_none_or(check_string))
+    if customer_id:
         # Running into the 60 character topic limit.
         # topic = '[{}](https://dashboard.stripe.com/customers/{})' % (customer_id, customer_id)
         topic = customer_id
     body = None
 
     def update_string(blacklist: Sequence[str] = []) -> str:
         assert "previous_attributes" in payload["data"]
-        previous_attributes = payload["data"]["previous_attributes"]
-        for attribute in blacklist:
-            previous_attributes.pop(attribute, None)
+        previous_attributes = {
+            key: value
+            for key, value in payload["data"]["previous_attributes"].items()
+            if key not in blacklist
+        }
```

Comment:
Any reason for changing this logic? (Just curious, not suggesting to revert)

Reply:
`previous_attributes` was recognized as a WildValue and that raised a Mypy error since WildValue does not have the `pop` method. That was the reason for the change.

Reply:
Let's try to keep logic changes like this in prep commits separate from the WildValue migrations themselves; it makes reviewing a lot harder if there's other things happening in a commit than just switching to WildValue.